v5_Gesture_Recognition_Serving.py

```diff
@@ -0,0 +1,268 @@
+#!/usr/bin/env python3
+"""Simple Flask example to serve Keras model."""
+
+import glob
+import json
+import os
+
+import numpy as np
+import pandas as pd
+
+from joblib import load
+from pathlib import Path
+
+from flask import Flask
+from flask import abort
+from flask import request
+
+from flask_restplus import fields
+from flask_restplus import Resource, Api, Namespace
+
+from http import HTTPStatus
+
+import tensorflow as tf
+
+from sklearn.preprocessing.label import LabelEncoder
+from tensorflow import keras
+
+from demo_lib.demo import process_motion_rotation
+from demo_lib.demo import clean_data
+from demo_lib.demo import featurize
+
+
+_HERE = Path(__file__).parent
+_EXAMPLE_DATA_PATH = Path(_HERE, 'data/dance/floss/example.json')
+
+_MODEL_DIR = Path(os.getenv("MODEL_DIR", _HERE / "models/v4"))
```

Comment:
Should be v5?

Reply:
And here I thought I hunted down all of the damn `v4`s. Should have really stuck to git-only versioning. Thanks!